packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -150,6 +210,12 @@ class BpkBannerAlert extends Component {
         </BpkAnimateHeight>
       </section>
     );
+
+    return (
+      <AnimateAndFade className={className} animateOnEnter={fadeIn} animateOnLeave={dismissable} show={shown}>
+        {bannerAlert}
+      </AnimateAndFade>
+    );
     /* eslint-enable */
```

Comment:
Not sure why this one keeps popping up, are we disabling es-link at some point in this component?

Reply:
Yup - that was already done here before - https://github.com/Skyscanner/backpack/blob/fd973903db3066106917c45660133537ae36ab7f/packages/bpk-component-banner-alert/src/BpkBannerAlert.js#L182

It was disabled as es-lint is picking up on accessibility issues with the touchable highlight, but the touchable highlight is "inaccessible" by design as the chevron button provides the necessary accessibility 😸